File: DeliveryCrab/Models/Basket.cs
﻿using System.ComponentModel.DataAnnotations.Schema;
namespace DeliveryCrab.Models
{
    public class Basket
    {
        public int Id { get; set; }
        public int Userid { get; set; }
        public int Productid { get; set; }
        public string? Productname { get; set; }
        public float Price { get; set; }
        public int Count { get; set; }
    }
}

File: DeliveryCrab/ClientApp/src/app/models/basket.ts
export class Basket{
  constructor(
    public id?:number,
    public userid?:number,
    public productid?:number,
    public productname?:string,
    public price?:number|undefined,
    public count?:number|undefined,
    public cost?:number|undefined,
  ){}
}
